projects/ng-components/lib/datepicker/datepicker.component.ts

```diff
@@ -188,6 +188,7 @@ export class DatepickerComponent implements ControlValueAccessor {
     }
 
     // get and set accessor----------------------
+    @Input()
```

Comment:
This is not a custom input but the value you get with NgModule.

Reply:
you should remove it.

Reply:
Nope, this is actually a good practice, if you don't include those binding with `[value]` will through an error since Angular will assume that `value` doesn't exist in the component even though it does. Adding this decorator is only to inform Angular that it exists and you can bind to it.